### PR TITLE
::: 0.19.0-release Enhancements based on customer feedback.

### PR DESCRIPTION
**Pull request by SteveMutungi254**
0.19.0-prerelease Enhancements based on customer feedback.

### Acknowledgement:

@alexandair's YouTube video: https://youtu.be/PUh20lNUYas?si=DE8EwochYSAjOK8g

### CODE REVIEW

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupMember.md [hunk 1]
```diff
@@ -55,28 +55,32 @@ To list members of a hidden group, the `Member.Read.Hidden` permission is also r
 
 ## Examples
 
-### Example 1: Get a group member by ID
+### Example 1: Retrieve and Select Group Member Properties
 
 ```powershell
-Connect-Entra -Scopes 'GroupMember.Read.All'
```
Comment (alexandair) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupMember.md:61]: This line shouldn't be removed.

**Review by alexandair — CHANGES_REQUESTED** [review]

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupMember.md (hunk 1 above)
Comment (SteveMutungi254) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupMember.md:61]: Great catch! @alexandair. I will add it again.

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupMember.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ To list members of a hidden group, the `Member.Read.Hidden` permission is also r
 ```powershell
 Connect-Entra -Scopes 'GroupMember.Read.All'
 $group = Get-EntraBetaGroup -Filter "DisplayName eq 'Sales and Marketing'"
-Get-EntraBetaGroup -GroupId $group.Id | Get-EntraBetaGroupMember | Select-Object Id, DisplayName, '@odata.type'
+Get-EntraBetaGroup -GroupId $group.Id | Get-EntraBetaGroupMember | Select-Object Id, DisplayName, '@odata.type'
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupMember.md:63]:
```suggestion
Get-EntraBetaGroup -GroupId $group.Id | Get-EntraBetaGroupMember | Select-Object Id, DisplayName, '@odata.type'
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraGroupMember.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ To list members of a hidden group, the `Member.Read.Hidden` permission is also r
 ```powershell
 Connect-Entra -Scopes 'GroupMember.Read.All'
 $group = Get-EntraGroup -Filter "DisplayName eq 'Sales and Marketing'"
-Get-EntraGroup -GroupId $group.Id | Get-EntraGroupMember | Select-Object Id, DisplayName, '@odata.type'
+Get-EntraGroup -GroupId $group.Id | Get-EntraGroupMember | Select-Object Id, DisplayName, '@odata.type'
```
Comment (KenitoInc) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraGroupMember.md:63]:
```suggestion
Get-EntraGroup -GroupId $group.Id | Get-EntraGroupMember | Select-Object Id, DisplayName, '@odata.type'
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraGroupMember.md [hunk 4]
```diff
@@ -87,13 +87,13 @@ Get-EntraGroupMember -GroupId $group.Id -Top 2
...
-00aa00aa-bb11-cc22-dd33-44ee44ee44ee
-11bb11bb-cc22-dd33-ee44-55ff55ff55ff
+cccccccc-8888-9999-0000-dddddddddddd
+dddddddd-9999-0000-1111-eeeeeeeeeeee
 ```
 
 This example demonstrates how to retrieve top two groups from Microsoft Entra ID.  
 
-- `-GroupId` specifies the ID of a group.
+- `-GroupId` specifies the ID of a group.
```
Comment (KenitoInc) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraGroupMember.md:96]:
```suggestion
- `-GroupId` specifies the ID of a group.
```

**Pull request merged**